drivers/uktty/pl011/pl011.c

```diff
@@ -128,7 +128,7 @@ void pl011_console_init(const void *dtb)
 		UK_CRASH("Could not find proper size cells!\n");
 
 	regs = fdt_getprop(dtb, offset, "reg", &len);
-	if (regs == NULL || (len < (int)sizeof(fdt32_t) * (naddr + nsize)))
+	if (!regs || (len < (int)sizeof(fdt32_t) * (naddr + nsize)))
```

Comment:
Let's use `unlikely` on all `UK_CRASH` cases?

Reply:
Good point, I'll do that. Thanks 👍🏼